Makefile

```diff
@@ -38,3 +46,12 @@ build-server: swig
 
 clean: clean-sdk
 	rm -rf bin gen/gen_wrap.cxx gen/gen.go
+
+appimage: build-module
+	cd etc/packaging/appimages && BUILD_CHANNEL=${BUILD_CHANNEL} appimage-builder --recipe rplidar-module-`uname -m`.yml
+	cd etc/packaging/appimages && ./package_release.sh
```

Comment:
You can remove this line, since we are not tagging versions. (We only need to tag versions when using a stable release channel.)